apim/templates/ui-configmap.yaml

```diff
@@ -15,15 +15,15 @@ data:
   gzip-types: "*"
   constants.json: |
     {
-      "baseURL": "https://{{ index .Values.api.ingress.hosts 0 }}{{ .Values.api.ingress.path }}/",
+      "baseURL": "https://{{ index .Values.api.ingress.hosts 0 }}/management/",
```

Comment:
Why did you change this part?

Reply:
thank you for quick response. 
Wenn all pathes for ingress should be mapped from aws alb ingress controller, there must stay wildcard like  /*  , simple slash is not enough for this ingress controller. It is well known behaviour  see hier -> https://github.com/kubernetes-sigs/aws-alb-ingress-controller/issues/667
If i change api.ingress.path to /* than baseURL looks like https://api.example.com/*/ , what is not valid path.
Same for other changes.

Reply:
In that case, why not providing two distinct properties:
- gateway.ingress.path (for conf)
- gateway.ingress.exposed-path (for ingress)

WDYT?

@chicco785 any idea?

Reply:
i also thought about this possibility, it can be a little confusing because we don't use than path anywhere in ingress, only in configmaps. In default case ( no aws-alb-ingress-controller) these variables are identical.
But it is definitely better than hard-coded pathes.

Another possibility is to filter out wildcard from path in configmaps with replace.

Reply:
Having a new boolean property `wildcard` under `ingress`, at the same level as `path` would not be sufficient?

If `wildcard` is `true` then, set `/*` for all ingresses path, and use only `path` for configmap.

WDYT?

Reply:
sounds good, but can possibly lead to failures because developer doesn't expect that all endpoints are mapped from ingress controller and that regardless of the fact, that developer configured his path in ingress.apth. What do you think about this filter:
https://github.com/dmytrokosiachenko/gravitee-helm/pull/1/files

Reply:
@brasseld what do you think, please respond